crates/bevy_ecs/src/storage/blob_vec.rs

```diff
@@ -35,6 +35,8 @@ impl std::fmt::Debug for BlobVec {
 }
 
 impl BlobVec {
```

Comment:
What’s a blob vec and what is a blob?

Reply:
A Blob is usually just an arbitrary block of contiguous memory without any identity, and could be used to represent any arbitrary data (i.e. string, arrays, etc). This is an extendable and reallcatable blob, which makes it a blobby Vec, a BlobVec.

Reply:
Why not throw that comment in the docs?